sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -742,6 +742,16 @@ def test_gaussian_mixture_verbose():
             sys.stdout = old_stdout
 
 
+def test_gaussian_mixture_n_iter():
```

Comment:
Maybe merge both test function to a pytest parametrized test under `sklearn/mixture/tests/test_mixture.py`

Reply:
Or actually somewhere under common tests and run it only for these two estimators for now?